home.css

```diff
@@ -0,0 +1,45 @@
+* {
+  scroll-behavior: smooth;
+}
+.landing-page {
+  display: flex;
+  height: 100vh;
+  justify-content: space-around;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 1rem;
+  background: var(--lightpurple2);
+}
+
```

Comment:
You can add the styling to fix the image for landing page as :
`landing-page__img img {
    max-width : 100%;
    width: 600px;
}`

Reply:
Thanks for the feedback , Have added the specified class to home.css, the landing page svg looks responsive and bigger now